app/src/main/java/com/x8bit/bitwarden/data/auth/datasource/network/service/IdentityServiceImpl.kt

```diff
@@ -127,4 +129,20 @@ class IdentityServiceImpl(
             .sendVerificationEmail(body = body)
             .map { it?.content }
     }
+
+    override suspend fun verifyEmailRegistrationToken(
+        body: VerifyEmailTokenRequestJson,
+    ): Result<VerifyEmailTokenResponseJson> = unauthenticatedIdentityApi
+        .verifyEmailToken(
+            body = body,
+        ).map {
+            VerifyEmailTokenResponseJson.Valid
```

Comment:
Does it work if you change the return type on the API from `Unit` to `VerifyEmailTokenResponseJson.Valid`?

Reply:
I was seeing a serialization exception

Reply:
I was facing the same issue it needs to be Unit and then map it afterwards